vars/container.groovy

```diff
@@ -0,0 +1,307 @@
+// TODO: Add AWS_SSH_PEM_KEY_NAME, AWS_SSH_RSA_KEY_NAME, CONFIG_NAME to the creds
```

Comment:
You're adding this where?

Reply:
Naming across secrets isn't standard, this is for that, we will rename or add new secrets for these values